### DISCUSSION TITLE
How to create REFCLASSIF file, when run SQANTi3 Rescue

### DISCUSSION THREAD

**Discussion by MR-D-CJ** [ConesaLab/SQANTI3#350] [General] [answered]
When `sqanti.py rescue` is run, the `-k (--refClassif)` parameter exists, requiring input of the reference SQANTi3 QC result file. For the production of this file, the following three ways are correct, the code is as follows: 

**according to the transcript sequence**
`python /PATH/SQANTI3-5.2.2/sqanti3_qc.py \
/PATH/GRCh38/genecode/gencode.v47.transcripts.fa \
/PATH/GRCh38/genecode/gencode.v46.annotation.gtf \
/PATH/GRCh38/GCA_000001405.15_GRCh38_no_alt/GCA_000001405.15_GRCh38_no_alt_analysis_set.fa \
--CAGE_peak /PATH/polyA_info/human.refTSS_v3.1.hg38.sorted.bed \
--polyA_motif_list /PATH/polyA_info/mouse_and_human.polyA_motif.txt \
-o GCA_000001405.15_GRCh38_no_alt \
-d /PATH/SQANTi3_ref/GRCh38 \
--fasta \
--force_id_ignore \
--cpus 20 --report html`

**according to the transcript annotation file**
`python /PATH/SQANTI3-5.2.2/sqanti3_qc.py \
/PATH/GRCh38/genecode/gencode.v47.transcripts.gtf \
/PATH/GRCh38/genecode/gencode.v46.annotation.gtf \
/PATH/GRCh38/GCA_000001405.15_GRCh38_no_alt/GCA_000001405.15_GRCh38_no_alt_analysis_set.fa \
--CAGE_peak /PATH/polyA_info/human.refTSS_v3.1.hg38.sorted.bed \
--polyA_motif_list /PATH/polyA_info/mouse_and_human.polyA_motif.txt \
-o GCA_000001405.15_GRCh38_no_alt \
-d /PATH/SQANTi3_ref/GRCh38 \
--fasta \
--force_id_ignore \
--cpus 20 --report html`

**according to the genome annotation file**
`python /PATH/SQANTI3-5.2.2/sqanti3_qc.py \
/PATH/GRCh38/genecode/gencode.v46.annotation.gtf \
/PATH/GRCh38/genecode/gencode.v46.annotation.gtf \
/PATH/GRCh38/GCA_000001405.15_GRCh38_no_alt/GCA_000001405.15_GRCh38_no_alt_analysis_set.fa \
--CAGE_peak /PATH/polyA_info/human.refTSS_v3.1.hg38.sorted.bed \
--polyA_motif_list /PATH/polyA_info/mouse_and_human.polyA_motif.txt \
-o GCA_000001405.15_GRCh38_no_alt \
-d /PATH/SQANTi3_ref/GRCh38 \
--fasta \
--force_id_ignore \
--cpus 20 --report html`

which one is true?

**Comment by lkondratova** [comment, marked as answer]
Hi @MR-D-CJ,
For this step, you should compare the annotation file against itself. Assuming you used gencode.v46.annotation.gtf for SQANTI3 on your samples, the first two arguments should be gencode.v46.annotation.gtf gencode.v46.annotation.gtf (as stated in your third option).
Please let us know if you have any further questions.
